docs/open-source/best-practices.mdx

```diff
@@ -0,0 +1,132 @@
+can't work out how to install or use your project, they'll give up and move on.
+This page covers licensing, packaging, testing, and documenting your project,
+all of which improve your users' experience. If you're releasing research code,
+following best practices will also help others reproduce your results.
+
+Once you've polished it up, consider adding your project to the [Qiskit
+ecosystem](https://www.ibm.com/quantum/ecosystem).
+
+## GitHub
+
+Most Qiskit projects are hosted on [GitHub](https://github.com/), a source-code
+hosting platform. GitHub is based on [Git](https://git-scm.com/) (a version
+control system) and provides a large set of tools to help collaborate on your
+project and keep it to a high quality. See [Github
+skills](https://skills.github.com/) to get up to speed on GitHub quickly.
+
+Some key features are:
+
+* **Issues and Pull requests**
+
+  Users can use GitHub to report bugs in your project or request changes to it.
+  Your team and external contributors can use pull requests to propose and
+  review changes to your project's code.
+
+  If you want to accept contributions from others, make sure to add a
+  [contributing
+  guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
+  and a [code of
+  conduct](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project).
+
+* **GitHub actions**
+
+  This feature runs scripts when certain events happen in your GitHub repo. For
+  example, you can run your [test suite](#test-your-project) when you push new
+  code or automatically [publishing your package](#package-your-project) when
+  you tag a commit.
+
+* **Security features**
+
+  GitHub supports features to keep your projects secure. These include:
+
+  * [Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates)
+    – a tool to automatically update your dependencies.
+  * [Trusted publishing to PyPI](https://docs.pypi.org/trusted-publishers/) –
+    to reduce the risk of a malicious actor gaining control of your package.
+  * [Branch
+    protection](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
+    – to make sure code meets some criteria before being pushed to certain
+    branches.
+
+The rest of this page will assume you have your project hosted on GitHub.
+
+## Choose a license
+
+If you want others to use your project, you *must* choose an appropriate
+license. Otherwise, your grant no permission for others to use it. Most
+projects in the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) use
+the Apache 2.0 or MIT licenses. Visit GitHub's
+[choosealicense.com](https://choosealicense.com) to decide which license is
+best for you.
```

Comment:
cc @1ucian0. I wonder if we should have a disclaimer similar in spirit to what American financial advice blogs say:

> Big Bank and its affiliates do not provide tax, legal or accounting advice. This material has been prepared for informational purposes only, and is not intended to provide, and should not be relied on for, tax, legal or accounting advice. You should consult your own tax, legal and accounting advisors before engaging in any transaction.

```suggestion
best for you, and consider seeking legal help.
```